weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PromptPage/PromptTab.tsx

```diff
@@ -0,0 +1,58 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import Accordion from '@mui/material/Accordion';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import Typography from '@mui/material/Typography';
+import classNames from 'classnames';
+import React from 'react';
+
+import {LoadingDots} from '../../../../../LoadingDots';
+import {Tailwind} from '../../../../../Tailwind';
+import {usePrompt} from './hooks';
+import {MessageList} from './MessageList';
+import {PlaceholdersPanel} from './PlaceholdersPanel';
+import {Data} from './types';
+
+type PromptTabProps = {
+  entity: string;
+  project: string;
+  data: Data;
+};
+
+export const PromptTab = ({entity, project, data}: PromptTabProps) => {
+  const {loading, prompt} = usePrompt(entity, project, data);
+  if (loading) {
+    return <LoadingDots />;
+  }
+  if (!prompt) {
+    return null;
+  }
+
+  const hasPlaceholders = prompt.placeholders.length > 0;
```

Comment:
I am not seeing placeholders in the ui, for example here: https://beta.wandb.ai/jamie-rasmussen/prompt-demo/weave/objects/travel_agent/versions/hawXYs4DrtroeANocVIc55PYRziAJXoRjFfmXpn71fc

Reply:
Yeah, I think I broke this when I decided it would be better for the placeholders to be recomputed than stored as part of the object. Might be for the best, the accordion material UI component I put in looked really ugly.